### PR TITLE
Updated the definition of scale_color_continuous

### PR DESCRIPTION
**Pull request by hlendway**
Updated the definition of scale_color_continuous to be scale_colour_continous instead of gradient so that it functions as the documentation states. Fixes issue #3053

### CODE REVIEW

**Comment by clauswilke** [comment]
Please add a news item to https://github.com/tidyverse/ggplot2/blob/master/NEWS.md

**Comment by hadley** [comment]
@clauswilke I have a github snippet for news bullets:

Can you please add a bullet to NEWS? It should briefly [describe the change](http://style.tidyverse.org/news.html) and end with `(@yourname, #issuenumber)`.

**Comment by hlendway** [comment]
I will add to NEWS.md today.

File: NEWS.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # ggplot2 3.1.0.9000
+  
+* `scale_color_continuous()` now points at `scale_colour_continuos()` so that it 
+  will handle `type` = "viridis" as the documentation states (@hlendway, #3079).
```
Comment (clauswilke) [inline NEWS.md]: The closing backtick should be after `"viridis"`, as in `` `type = "viridis"` ``.

**Comment by clauswilke** [comment]
This is good now. Unfortunately there is a merge conflict with the latest version of ggplot2, due to the many changes that have happened in `NEWS.md`. You'll have to merge with upstream to resolve this. Here is a rough explanation of how this works: https://help.github.com/articles/syncing-a-fork/ You'll have to do this in your development branch instead of master, though. (In step 4, instead of checking out master, you check out the branch you're working on. Everything else is the same.)

**Comment by hlendway** [comment]
Thanks @clauswilke for the instructions, hopefully that worked OK, please let me know if there are additional things to do.  I really appreciate the guidance through the process!

**Comment by BangerNPoo** [comment]
I think u need to stay out my husband Gmail... til we are divorced!! please!!

**Pull request merged**